### DISCUSSION TITLE
WinXP/Win7-style Start button icon?

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#891] [Ideas]
Since I use OpenShell and Classic theme, this function does not work for me anyway, but for perfection, a Win7-style start button icon may be added, it is located in %SystemRoot%\System32\slui.exe, 2 (an icon-type resource).

**Comment by valinet** [comment]
Thanks, will think about it. At least I know where to get it from, pretty useful.
